modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gnirs/GnirsControllerEpics.scala

```diff
@@ -226,6 +226,12 @@ object GnirsControllerEpics extends GnirsController {
     _ <- GnirsEpics.instance.abortCmd.post
   } yield ()
 
+  private def removePartName(s: String) = {
```

Comment:
Why this?

Reply:
If you read the current filter, GNIRS gives you something like H_G5534. I need to remove the trailing code before comparing.